src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  position: relative;
+  width: 7 * ($cell-size + $borderWidth) + 2 * $calendarPadding - $cellGap;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cellGap;
+
+  @each $day-name, $day-index in $days {
+    &--start-day-#{$day-name} {
+      :nth-child(1) {
+        margin-left: ($cell-size + $cellGap) * ($day-index - 1);
+      }
+    }
+  }
+
+  &__date {
+    height: $cell-size;
+    width: $cell-size;
+    gap: $cellGap;
+    background-color: #eee;
+    border: $borderWidth solid black;
+    transition-duration: $animationDuration;
+
+    @for $dayNumber from 1 through 31 {
+      &:nth-child(#{$dayNumber})::before {
+        content: "#{$dayNumber}";
+        display: flex;
+        justify-content: space-around;
+        align-content: space-between;
+        font-size: $dayFontSize;
+        margin-top: 35px;
```

Comment:
```suggestion

```
You can move these properties from your loop to the regular '&__date::before' selector